app/frontend/src/styles/application/govuk-error-summary.scss

```diff
@@ -0,0 +1,8 @@
+.govuk-error-summary__list {
+  li {
```

Comment:
I don't think I can apply classes to these `li`s because they exist inside the govuk-error-summary, which is html from a gem, not from any of our templates.

Reply:
oh i didnt appeciate it was gov component. to me seems odd they wouldnt put classes on everything contained but maybe they came to same conclusion i mentioned above thats its pretty obvious whats going to be inside something called `__list`. id still say the `<a>` is a bit more debatable but its part of the thing so cool